frontend/webservice/workgroup_manage.cgi

```diff
@@ -556,9 +556,13 @@ sub remove_acl {
         return;
     }
 
+    my ($result,$error) = $db->execute_query("select interface_id, workgroup_id from interface_acl where interface_acl_id = $interface_acl_id");
```

Comment:
BAD!!! Use this instead:
```
$db->execute_query("select interface_id, workgroup_id from interface_acl where interface_acl_id=?", [$interface_acl_id]);
```
See https://www.hacksplaining.com/prevention/sql-injection for more details

Reply:
Ended up deleting that query since the result was only being used by the OESS::DB::ACL::remove for the workgroup_id but the ACL remove method doesn't even use the workgroup_id or the interface_id.